src/Client/Suggest.cpp

```diff
@@ -101,7 +101,9 @@ static String getLoadSuggestionQuery(Int32 suggestion_limit, bool basic_suggesti
         add_column("name", "columns", true, suggestion_limit);
     }
 
-    query = "SELECT DISTINCT arrayJoin(extractAll(name, '[\\\\w_]{2,}')) AS res FROM (" + query + ") WHERE notEmpty(res)";
+    /// FIXME: Forbid this query using new analyzer because of bug https://github.com/ClickHouse/ClickHouse/pull/50430#issuecomment-1576860893
```

Comment:
Also, it requires creating a separate issue because otherwise everybody will forget about it and it won't be fixed.

Reply:
I just found it might be another bug other than it ... T_T

Reply:
It's unclear why it's related to the analyzer. If we merge this change as is we'll definitely forget about it until we try to remove the code of the old analyzer.